frontend/src/lib/api.ts

```diff
@@ -842,6 +842,10 @@ class ApiRequest {
         return this.dataWarehouseSavedQueries(teamId).addPathComponent(id)
     }
 
+    public dataWarehouseSavedQueryActivity(id: DataWarehouseSavedQuery['id'], teamId?: TeamType['id']): ApiRequest {
+        return this.dataWarehouseSavedQuery(id, teamId).addPathComponent('activity')
```

Comment:
Note sure how team_id is optional here, but I suppose it's not too big of a deal.